crates/ruma-events/src/secret_storage.rs

```diff
@@ -0,0 +1,5 @@
+//! Module for events in the *m.secret_storage* namespace.
+
+pub mod default_key;
+pub mod key;
+pub mod some_secret;
```

Comment:
I can't find this in the v1.2 spec which includes secret storage. It also seems like it was just a placeholder originally (based on the name). This should probably be removed.